python/lsst/meas/extensions/trailedSources/NaivePlugin.py

```diff
@@ -296,13 +324,53 @@ def findLength(self, Ixx, Iyy):
         # c_min is given by the case: Ixx == Iyy, ie. a point source.
         # c_max is given by the limit Ixx >> Iyy.
         # Emperically, 0.001 is a suitable lower bound, assuming Ixx > Iyy.
-        z, results = sciOpt.brentq(lambda z: self._computeSecondMomentDiff(z, c),
+        z, results = sciOpt.brentq(lambda z: cls._computeSecondMomentDiff(z, c),
```

Comment:
Which variables are you referring to? `z` and `c`?

Reply:
I noticed it here specifically, but in general too, it may seem like overkill often, but it helps someone else get up to speed. It is a bit of a discretion thing, but our dev guide does say when possible prefer descriptive names.

Reply:
Gotcha. I agree in general. Here, I'm not sure how to do that, since `z` and `c` are transformed variables and don't really have a physical meaning. I just used the same notation I am using in the paper.

Reply:
Ok, that is why I asked, there is certainly a carve out for things like that, commonly you just reference the paper equation describing it in a comment or something like that. Even easier if you put the citation in a Note section in the task docstring, and then the comment can just be something like `# variables c and z as described in eq X`